content/WebAuthn/Concepts/Authenticator_Management/Use_Cases_and_Scenarios.adoc

```diff
@@ -0,0 +1,117 @@
+* You want your users to utilize anything from FIDO2 security keys, to copyable passkeys, to U2F authenticators.
+
+==== Implementation Guidance
+
+Below is a link to the Yubico WebAuthn Starter Kit. This will provide a full reference architecture where you will deploy a live WebAuthn application using AWS. The Starter Kit aims to teach you how to:
+
+* Develop a sample end-to-end WebAuthn application
+* Integrate a WebAuthn application with a common identity provider
+* Understand UX and UI best practices and considerations
+* Manage WebAuthn credentials through an account lifecycle
+
+link:https://developers.yubico.com/Developer_Program/WebAuthn_Starter_Kit/[Yubico WebAuthn Starter Kit]
+
+==== Vulnerability remediation
+
+There is another scenario when allowing all FIDO2 authenticators to be used in your environment; what do you do if an authenticator is found as vulnerable? In a high risk controlled environment you may opt to reject the authenticator entirely, but you may want to take a different approach for a consumer facing application. 
+
+In this scenario we recommend to send alerts to account holders who may have a credential registered that was created with an authenticator with a vulnerability. The guidance linked below will give examples on how to identify vulnerable authenticators in your environment, and how to alert users on how they can self remediate, if they choose to do so.
+
+link:/WebAuthn/Concepts/Authenticator_Management/Implementation_Guidance/Vulnerability_Remediation.html[Vulnerability remediation implementation guidance] 
+
+=== Limit authenticators based on device characteristics/features
+
+This scenario is where you begin to add controls to make your application dynamically respond to different authenticators. This will allow you to add a higher degree of security to user accounts, while not constraining account holders to specific authenticators. Understanding this section will be critical for implementing link:https://developers.yubico.com/Developer_Program/WebAuthn_Starter_Kit/Adaptive_Multi-Factor_Authentication.html[adaptive multi factor authentication] into your application; your relying party will be able to adapt and route users across different flows based on the risk profile of the authenticator used, and the action being attempted by the user. 
+
+It’s worth noting that this category is where you will begin to introduce friction for users who are not leveraging high assurance devices. A user who is leveraging a high assurance device will be able to seamlessly authenticate due to the very high degree of confidence that their request is legitimate. Authentication ceremonies performed with lower assurance devices may require another factor to authenticate, or step-up authentication to perform certain actions. 
+
+This scenario works best when:
+
+* A user's authenticator is missing user verification
+* A user is attempting to use a copyable passkey in a sensitive environment
```

Comment:
Consider rewording to something like:
* A user has previously authenticated with a lower assurance credential (e.g. copyable passkey) and has requested an action to access or modify sensitive data.